### PR TITLE
Update spring boot example

### PR DESCRIPTION
**Pull request by bertschneider**

### CODE REVIEW

File: spring-boot-example/cifuzz.yaml [hunk 1]
```diff
@@ -22,9 +22,8 @@
 
 ## Command-line arguments to pass to libFuzzer.
 ## See https://llvm.org/docs/LibFuzzer.html#options
-engine-args:
- - -rss_limit_mb=4096
- - --instrumentation_includes=com.example.**
+#engine-args:
```
Comment (kyakdan) [inline spring-boot-example/cifuzz.yaml:25]: Why are these options deleted?

File: spring-boot-example/.mvn/wrapper/maven-wrapper.properties [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```
Comment (kyakdan) [inline spring-boot-example/.mvn/wrapper/maven-wrapper.properties:1]: Is this autogenerated?

File: spring-boot-example/mvnw [hunk 3]
```diff
@@ -8,7 +8,7 @@
 # "License"); you may not use this file except in compliance
 # with the License.  You may obtain a copy of the License at
 #
```
Comment (kyakdan) [inline spring-boot-example/mvnw:10]: Why have these scripts changed?

Comment (bertschneider) [reply spring-boot-example/mvnw:10]: Executing `./mvnw` did not work, as the required file in `.mvn` was missing. I regenerated the wrapper script with `mvn` version `3.8.4` using `mvn wrapper:wrapper`, which also changed the existing files.

File: spring-boot-example/.mvn/wrapper/maven-wrapper.properties (hunk 2 above)
Comment (bertschneider) [reply spring-boot-example/.mvn/wrapper/maven-wrapper.properties:1]: Yes, autogenerated by the wrapper script. This file is required to execute `./mvnw`.

File: spring-boot-example/cifuzz.yaml (hunk 1 above)
Comment (bertschneider) [reply spring-boot-example/cifuzz.yaml]: `rss_limit_mb` is set dynamically by Jazzer based on the available heap size and should not be set manually. 
The new Jazzer version should use the project structure to automatically set the instrumentation filer. Apparently that does not work for this project so I added the flag back in.

Comment (kyakdan) [reply spring-boot-example/cifuzz.yaml]: Thanks! the remote run works.

**Pull request merged**